src/main/java/duke/task/TestTask.java

```diff
@@ -0,0 +1,21 @@
+package duke.task;
+
+public class TestTask {
+    private String description;
+    private boolean isDone;
+    private String deadline;
+    private boolean needToDo;
+
+    public String getDescription() {
+        return description;
+    }
+
+    public String getStatusIcon() {
```

Comment:
There are generally few or no comments in each of your files. Refer to [javadocs](https://se-education.org/guides/conventions/java/basic.html#comments) for examples on what to include to explain what the method is doing.